notmuch/config.c

```diff
@@ -110,6 +110,10 @@ static struct ConfigDef NotmuchVars[] = {
   { "nm_query_type", DT_STRING, IP "messages", 0, NULL,
     "(notmuch) Default query type: 'threads' or 'messages'"
   },
+  {
+    "nm_query_window_additional_terms", DT_STRING, 0, 0, NULL,
+    "(notmuch) additional notmuch search terms for query windows to match on instead of the date."
+  },
```

Comment:
Maybe:
"(notmuch) additional notmuch search terms for messages to be shown regardless of date."

Reply:
That's an improvement. I'm happy to replace this version.

Though I showed both descriptions to a friend and they were still confused about what the variable did. Should we take a more direct approach in the documentation and say something along the lines of:
"(notmuch) convert window to "date:... or (additional terms)" statement."

Reply:
maybe with an example of what the term looks like after expansion?

>For example, using the mailbox notmuch://?query=tag:inbox as the
mailbox and tag:flagged and tag:unread as the additional search terms
will produce a query window such as:
>
>notmuch://?query=tag:inbox and (date:... or (tag:flagged and tag:unread))


Made it very obvious

Reply:
> maybe with an example of what the term looks like after expansion?

That's a great idea! I'll incorporate that into the PR when I'm back at my dev machine.